src/runners/violation_suppressions_runner.py

```diff
@@ -54,7 +55,7 @@ def run_suppression(squelch_name):
     print(f"Suppression query {squelch_name} executed")
 
 
-def main():
+def main(arg_optional = None):
```

Comment:
extra spaces

Reply:
Removed

Reply:
> some of those Any seem vestigial, but lgtm overall -- definitely install a PEP8 formatter on save, no need for us to review whitespace compliance :)

True! But had to add them for the mypy errors, will do! : )